subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -349,8 +350,9 @@ int ull_scan_reset(void)
 	return 0;
 }
 
-void ull_scan_params_set(struct lll_scan *lll, uint8_t type, uint16_t interval,
-			 uint16_t window, uint8_t filter_policy)
+uint32_t ull_scan_params_set(struct lll_scan *lll, uint8_t type,
+			     uint16_t interval, uint16_t window,
+			     uint8_t filter_policy)
```

Comment:
Just FYI, the TSC changed the character limit to 100, removing the 80 recommendation